workflows/src/main/resources/migration/account-info-schema.changelog-init.xml

```diff
@@ -0,0 +1,48 @@
+<?xml version="1.1" encoding="UTF-8" standalone="no"?>
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+                   xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+                   xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.5.xsd">
+    <changeSet author="R3.Corda" id="create_account_info_table_id">
+        <createTable tableName="accounts">
+            <column name="identifier" type="UUID">
+                <constraints nullable="false"/>
+            </column>
+            <column name="name" type="VARCHAR(255)">
+                <constraints nullable="false"/>
+            </column>
+            <column name="host" type="NVARCHAR(255)">
+                <constraints nullable="false"/>
+            </column>
+            <column name="status" type="BOOLEAN">
```

Comment:
Two things here:
 - `status` is an enum in your JPA entity. I think by default enum values are persisted as their ordinal values (so you'd get 0, 1, 2...). As such I don't think boolean is the right type here. If you'd rather not store ordinal values then you could either have strings in your entities, or try using an `AttributeConverter`.
- This column is constrained to be not null by the migration script but isn't constrained by the JPA entity

Reply:
Column now not nullable. Amended the type - hopefully this works...